tests/src/test/java/alluxio/hadoop/FileSystemAclIntegrationTest.java

```diff
@@ -413,19 +419,102 @@ public void objectStoragePermissionTest() throws Exception {
     create(sTFS, fileA);
     Assert.assertTrue(sUfs.exists(PathUtils.concatPath(sUfsRoot, fileA)));
 
-    // Verify the owner, group and permission of UFS is not supported and thus returns default
+    // Without providing "alluxio.underfs.s3.canonical.owner.id.to.username.mapping", the default
```

Comment:
Should this test set the `alluxio.underfs.s3.canonical.owner.id.to.username.mapping` property to `null`?